app/src/main/res/drawable/edit_text_black_border.xml

```diff
@@ -2,12 +2,12 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
   <stroke
     android:width="1dp"
-    android:color="@color/black" />
-  <corners android:radius="6dp" />
+    android:color="@color/grey" />
```

Comment:
This is not supposed to be modified here and also, as name specify it is black border and is used in other places where it should be black.